packages/perseus-core/tsconfig-build.json

```diff
@@ -0,0 +1,19 @@
+{
+    "extends": "../tsconfig-shared.json",
+    "compilerOptions": {
+        "outDir": "./dist",
+        "rootDir": "src",
+        "paths": {
+            // NOTE(kevinb): We have to repeat this here because TS doesn't do
+            // intelligent merge of tsconfig.json files when using `extends`.
```

Comment:
This works? I didn't realize we could have comments in JSON, but maybe we have a processor that helps with that?